src/components/InngestCommHandler.ts

```diff
@@ -636,7 +636,7 @@ export class InngestCommHandler<
         },
       });
 
-    this._isProd = actions.isProduction ?? isProd(env);
+    this._isProd = actions.isProduction ?? isPlatformDeploy(env);
```

Comment:
As discussed elsewhere (cc @goodoldneon), keeping the `isProd()` check here would be ace!
```suggestion
    this._isProd = actions.isProduction ?? isProd(env) || isPlatformDeploy(env);
```